log/log.go

```diff
@@ -330,6 +331,19 @@ func (l *Log) GetBoleto(content interface{}, msgType string) {
 	})()
 }
 
+//handleMaskErros Handle de erro para funções internas para garantir que, caso ocorra algum problema interno
+//o conteúdo continuará sendo enviado, porém sem a devida máscara.
+
+func (l *Log) HandleMaskErrors(method string, messageType string, methodContent interface{}) {
+	if r := recover(); r != nil {
+		errorDescription := fmt.Sprintf("Recovering from panic during %s operation", method)
+		logMessage := fmt.Sprintf("to {BankName} (%s) | {Recipient} - %s", messageType, errorDescription)
+		errorString := fmt.Sprintf("%s", r)
```

Comment:
Opa, aqui talvez da para simplificar 😀 
```suggestion
		recoverError := fmt.Errorf("%s", r)
```

Reply:
😮 da hora